tests/tests-asn1c-compiler/18a-class-OK.asn1.-B

```diff
@@ -73,7 +65,18 @@ message Operatorsquare {
 // {operator-root}
 message Operatorroot {
     // @inject_tag: aper:"unique,"
-    int32 code = 1 [ json_name="&code"];
+    int32 code = 1 [(validate.v1.rules).int32.const = 3, json_name="&code"];
```

Comment:
the `&` in the json name could be a problem?

Reply:
No, it shouldn't be